xml/deployment_prep_zseries.xml

```diff
@@ -433,7 +450,7 @@ Install=tftp://<replaceable>USER</replaceable>:<replaceable>PASSWORD</replaceabl
     </varlistentry>
    </variablelist>
   </sect2>
-
+  <xi:include href="deployment_prep_zseries_prep_io_auto_config.xml"/>
```

Comment:
As final optimization, I could imagine adding `DeviceAutoConfig=1` to the first example parmfile further down at
https://documentation.suse.com/sles/15-SP2/single-html/SLES-deployment/#ex-appdendix-parm-ex1
and append a hint to its example title to become something along the lines of:
"Example 5.8: Parmfile for an Installation from NFS with VNC and AutoYaST and accepting I/O device auto-configuration to prevent an interactive question"
```
<screen>ramdisk_size=131072 root=/dev/ram1 ro init=/linuxrc TERM=dumb
instnetdev=osa osainterface=qdio layer2=1 osahwaddr=
pointopoint=&exampledomainip;.1 hostip=&exampledomainip;.2
nameserver=&exampledomainip;.3 DeviceAutoConfig=1
install=nfs://&exampledomainip;.4/SLES/SLES-12-Server/s390x/DVD1
autoyast=http://&exampledomainip;.5/autoinst.xml
linuxrclog=/dev/console vnc=1 VNCPassword=testing</screen>
```

Reply:
@steffen-maier : Thanks, I applied this suggestion as well.